.github/pull_request_template.md

```diff
@@ -0,0 +1,104 @@
+---
+name: Pull Request
+about: Create a pull request
+title: "[TITLE]: Brief description of PR"
+labels: ''
+assignees: ''
+
+---
+<!-- Please remove any unused sections.
+
+Note that anything written between these symbols will not appear in the actual, published PR. They serve as instructions for filling out this template. You may want to use the 'preview' tab above this textbox to verify formatting before submitting.
+-->
+
+## Summary
+### Description of the change(s) you made
+
+### Manual verification steps performed
+1. Step 1
+2. Step 2
+3. ...
+
+### Screenshots (if applicable)
+<!-- If not applicable, please delete this section -->
+
+### Does this introduce any tech-debt items?
+<!-- List anything that will need to be addressed later -->
+___
+## Reviewer guidance
+### How can a reviewer test these changes?
+<!-- If not applicable, please delete this section -->
+
+
+### Are there any risky areas that deserve extra testing?
+<!-- If not applicable, please delete this section -->
+
+
+## References
+<!--
+Additional, helpful things to add in this section:
+ * links to mockups or specs for new features
+ * links to the diffs for any dependency updates, e.g. in iceqube or the perseus plugin
+-->
+### Addresses PR #_insert PR number here_
+
+## Comments
+<!-- Additional comments may be added here -->
+
+----
+
+### Contributor's Checklist
+<!-- After saving the PR, come through to tick off completed checklist items. Delete any sections that are not applicable to your PR -->
+
+PR process:
+
+- [ ] PR has the correct target branch and milestone
+- [ ] PR has 'needs review' or 'work-in-progress' label
+- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
+- [ ] If this is an important user-facing change, PR or related issue the `CHANGELOG` label been added to this PR. Note: items with this label will be added to the [CHANGELOG](https://github.com/learningequality/studio/blob/master/CHANGELOG.md) at a later time
+- [ ] If this includes an internal dependency change, a link to the diff is provided
+- [ ] The `docs` label has been added if this introduces a change that needs to be updated in the [user docs](https://kolibri-studio.readthedocs.io/en/latest/index.html)?
+- [ ] If any Python requirements have changed, the updated `requirements.txt` files also included in this PR
+- [ ] Opportunities for using Google Analytics here are noted
+- [ ] Migrations are [safe for a large db](https://www.braintreepayments.com/blog/safe-operations-for-high-volume-postgresql/)
```

Comment:
unless things have changed, some of these items might refer to Kolibri-specific processes and labels

Reply:
I found those in the current Studio PR template. Do we need to update them? I'm not sure if this list is overly specific

Reply:
I removed lines 55-57, since those were more Kolibri-specific.